### DISCUSSION TITLE
Multi Factor Auth using TOTP

### DISCUSSION THREAD

**Discussion by 274188A** [abpframework/abp#8877] [Q&A] [answered]
I'm trying to explore using MFA in a Web Pages template (MVC net5.0)

I'm hoping to use an approach called **TOTP** which is possible as shown in this [article](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/mfa?view=aspnetcore-5.0)

Is this something on the roadmap or is it even doable now?
Thanks

**Comment by realLiangshiwei** [comment, marked as answer]
There is no plan, you can implement it yourself
